src/layouts/app/pages/PetVet/index.spec.js

```diff
@@ -0,0 +1,133 @@
+import { describe, expect, it } from 'vitest';
+import { render, screen } from '@testing-library/vanilla';
+import { userEvent } from '@testing-library/user-event';
+import dayjs from 'dayjs';
+import PetVetPage from './index';
+
+function formatDate(date) {
+  return date ? dayjs(date).format('MM/DD/YYYY') : '';
+}
+
+const mapVaccine = (vaccine) => ({
+  id: vaccine.id,
+  title: vaccine.title,
+  date: formatDate(vaccine.date),
+  veterinary: vaccine.veterinary,
+});
+
+describe('PetVetPage', () => {
+  const mockVaccines = [
+    {
+      id: '1',
+      veterinary: 'Dr octopus',
+      title: 'Antirrábica',
+      date: new Date().toISOString(),
+    },
+    {
+      id: '2',
+      veterinary: 'Dr Felipa',
+      title: 'Raiva',
+      date: new Date(2023, 5, 2).toISOString(),
+    },
+    {
+      id: '3',
+      veterinary: 'Dr octopus',
+      title: 'Raiva',
+      date: new Date(2023, 2, 2).toISOString(),
+    },
+  ];
+  const argsWithVaccine = { vaccines: mockVaccines };
+  const argsWithVaccineEmpty = { vaccines: [] };
+  const renderPetVetPage = (parameters) => render(new PetVetPage(parameters));
+
+  it('renders the main container', () => {
+    renderPetVetPage(argsWithVaccineEmpty);
+    const title = screen.getByText('Conte-nos um pouco mais do seu animal');
+
+    expect(title).toBeInTheDocument();
+  });
+
+  describe('radios form', () => {
+    it('confirms if the neutering selection is correct', async () => {
```

Comment:
Oq correto é?

Correto é uma palavra abstrata, seu teste esta verificando se o neutering foi clicado no not neutered e é isso que a descrição do seu teste deve dizer!

Inclusiva não precisa checar se o true não ta selecionado, acredito que esse possa ser um outro teste de não permitir que 2 itens estejam selecionados ao mesmo tempo